tools/launch_openocd_gdb.sh

```diff
@@ -15,26 +15,26 @@ if [ $PLATFORM == "linux" ]
 then
   $DEVICE_GDB $EXECUTABLE -ex "source $GDBINIT_PATH"
 else # For all other platforms
-  $OPENOCD/bin/$OPENOCD_EXECUTABLE  -s $OPENOCD/scripts/ \
-    -c "source [find interface/$DEBUG_DEVICE.cfg]" -f $OPENOCD_CONFIG &
+  # $OPENOCD/bin/$OPENOCD_EXECUTABLE  -s $OPENOCD/scripts/ \
+  #   -c "source [find interface/$DEBUG_DEVICE.cfg]" -f $OPENOCD_CONFIG &
 
-  # Capture the pid of the background OpenOCD process
-  OPENOCD_PID=$(echo $!)
-  # Wait for OpenOCD to continue or quit
-  sleep 2
-  # Use kill to check if OpenOCD is running
-  kill -0 $OPENOCD_PID &> /dev/null
-  # Capture success or failure of check above
-  OPENOCD_IS_RUNNING=$?
-  if [ $OPENOCD_IS_RUNNING -ne 0 ]
-  then
-    RED="\x1B[31;1m"
-    RESET="\x1B[0m"
-    echo
-    echo -e "$RED OpenOCD failed to start, exiting $RESET"
-    echo
-    exit 1
-  fi
+  # # Capture the pid of the background OpenOCD process
+  # OPENOCD_PID=$(echo $!)
+  # # Wait for OpenOCD to continue or quit
+  # sleep 2
+  # # Use kill to check if OpenOCD is running
+  # kill -0 $OPENOCD_PID &> /dev/null
+  # # Capture success or failure of check above
+  # OPENOCD_IS_RUNNING=$?
+  # if [ $OPENOCD_IS_RUNNING -ne 0 ]
+  # then
+  #   RED="\x1B[31;1m"
+  #   RESET="\x1B[0m"
+  #   echo
+  #   echo -e "$RED OpenOCD failed to start, exiting $RESET"
+  #   echo
+  #   exit 1
+  # fi
```

Comment:
Undo all of this.

Reply:
Please try and update your laptop to see if that patches the openocd file location problem

Reply:
This has not been resolved. Please fix this.

Reply:
I un-comented the lines of code. However, ill need to play with this more.
I actually went back to WSL 1 because some of the commands (in contrast to the purpose of WSL 2) were slower and actually started to annoy me.
Its pretty quick to switch between the two of them "wsl --set-version Ubuntu 2" or "wsl --set-version Ubuntu 1"